docs/manual-testing.md

```diff
@@ -0,0 +1,39 @@
+Martha Manual Test
```

Comment:
```suggestion
Martha manual end-to-end test in a production-like environment
```
Probably good to explain exactly what we're testing (assuming the suggestion is correct)

Reply:
The test is supposed to be run in non-prod and prod, so maybe just "Martha Manual End-to-End Test"?

Reply:
"production-like environment" is a code word (possibly invented and used only by me) to refer to `dev`, `alpha`, `prod`, etc. – things that look like prod Terra but may or may not be it. Feel free to drop the term if you don't find it useful.

Reply:
Ok, I changed it; hope it accomplishes the spirit (if not the exact words) of your suggestion.